src/login.js

```diff
@@ -55,59 +56,71 @@ async function login (options) {
 
   return new Promise((resolve, reject) => {
     let spinner
-    if (!bare) {
-      // stderr so it is always visible
-      spinner = ora()
-      spinner.stopAndPersist({ text: 'Visit this url to log in:\n' + uri })
-      spinner.start('Waiting for browser login')
-    }
-    if (autoOpen) {
-      open(uri, { app })
-    }
 
-    const timerId = setTimeout(() => {
+    if (ciInfo.isCI && !bare) {
+      // CI path and !bare (so show spinner messages)
+      spinner = ora()
+      spinner.fail('CI Environment: Interactive login not supported. Use service token/env vars for authentication.')
```

Comment:
I would link directly to https://github.com/adobe/aio-apps-action here to show customers how to use technical account tokens / env vars